frontend/src/views/Myprofile.vue

```diff
@@ -0,0 +1,18 @@
+<script>
+import Profile from "../components/Profile.vue";
+
+export default {
+  components: { Profile },
+
+  mounted() {
```

Comment:
this function can be removed, it is not used anywhere, in fact, even the js file has been already removed from the assets folder 🙂